### ISSUE TITLE
scipy.weave not found

### ISSUE THREAD

**Issue by leviboyles** [JasperSnoek/spearmint#70]
scipy.weave is not found when doing "from spearmint import gp"  It seems weave is now a separate module from scipy

**Comment by bfeif** [comment]
yes. you can do a replace all, replacing 'scipy.weave' with 'weave', and everything works (so far). Of course you'll have to also install weave.

**Comment by FelixNeutatz** [comment]
alternatively, you can switch to an old version of scipy: python -m pip install scipy==0.18.1
